test/overall/plyqlParser.mocha.coffee

```diff
@@ -139,6 +152,8 @@ describe "SQL parser", ->
         SUM(`wiki`.`added`) / 4 AS TotalAddedOver4,
         NOT(true) AS 'False',
         -SUM(added) AS MinusAdded,
+        ABS(MinusAdded) AS AbsAdded,
+        ABSOLUTE(MinusAdded) AS AbsoluteAdded,
```

Comment:
It could also be fun (and almost no work to add EXP(X) https://dev.mysql.com/doc/refman/5.7/en/mathematical-functions.html#function_exp to PlyQL) basically EXP(X) => E.power(X) where E is 2.718281828459045